apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -484,9 +486,24 @@ const StyledLink = styled("a", {
   },
 });
 
+type DeployTarget = "vercel" | "netlify";
+
+const DeployTargets: Record<DeployTarget, string> = {
+  vercel: "npx vercel",
+  netlify: "npx netlify",
+};
+
+const DeployTargetDocs: Record<DeployTarget, string> = {
```

Comment:
Move everything into a single place:
```
deployTargets = {
  vercel: {
   command: ''
   target: '''
  }
}
```

Reply:
Try to not use `Record<>` without `"noUncheckedIndexedAccess": true` in tsconfig its super error prone.
declare `as const`
```ts
const deployTarget = { .... } as const;
```